library/ts-rollup/dev/static/css/main.css

```diff
@@ -1,6 +1,38 @@
+.flex-column-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+}
```

Comment:
I would have left this CSS part to the candidate
and I don't want to maintain/backport this in the ES-webpack example

Reply:
I just put a little CSS to order as minium the information on the page.
For the rest, it's to the challenger to change.

Reply:
I already backport all the change on the webpack challenge 😉  #14

Reply:
hehehe, you are quick
eventually everyone is free to change the design if they would like to